pkgs/misc/lssecret/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, pkg-config
+, libsecret
+, fetchFromGitLab
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  name = "lssecret";
+  version = "722013dc";
+
+  src = fetchFromGitLab {
+    owner = "GrantMoyer";
+    repo = name;
+    rev = version;
+    hash = "sha256-KLtxRXySoL9hT/c+O2arVR68ZYz+6Ce72YYCz60yot4=";
+  };
+
+  installPhase = "install -Dsm555 lssecret $out/bin/lssecret";
```

Comment:
```suggestion
  makeFlags = ["DESTDIR=$(out)"];
```

Reply:
See https://github.com/NixOS/nixpkgs/issues/65718 for the correct usage of DESTDIR

Reply:
I think this is different because its a single file, and the makefile doesnt mention `PREFIX` at all